gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -431,11 +431,8 @@
         <module name="JavadocMethod"> <!-- Java Style Guide: Where Javadoc is used -->
             <property name="scope" value="public"/>
             <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
```

Comment:
changes here are because javadoc check got split into two, this and `MissingJavadocMethod`. We actually only want this one that validates javadoc if it exists but doesn't enforce existence of javadoc